Client/Frontend/Browser/TabLocationView.swift

```diff
@@ -229,6 +230,11 @@ class TabLocationView: UIView {
         } else {
             urlTextField.text = url?.absoluteString
         }
+        // remove https:// (the scheme) from the url when displaying
+        if let scheme = url?.scheme {
+            let schemeString = "\(scheme)://"
+            urlTextField.text = url?.absoluteString.replacingOccurrences(of: schemeString, with: "")
```

Comment:
This will replace *all* occurrences. What if the URL contains something like `https://example.com/login?redirect=https://xyz.com/home`?

Reply:
good find!